linera-views/src/memory.rs

```diff
@@ -43,6 +43,10 @@ pub fn create_test_context() -> MemoryContext<()> {
 
 #[async_trait]
 impl KeyValueStoreClient for MemoryClient {
+    // For the memory container, memory is the limit and so usize::MAX
+    // could be used. But memory is intrisically a test framework and so
```

Comment:
typo: intrisically

Reply:
maybe use cfg(test) for a quick hack?

Reply:
Since memory was only used for tests, I thought it was better to put 100 for everything. But ok, let's do that.